Sources/App/CRP/JiraService+CRP.swift

```diff
@@ -1,17 +1,40 @@
-import Stevenson
 import Vapor
+import Stevenson
+
+extension JiraService {
+    func makeCRPIssue(
+        repoMapping: RepoMapping,
+        release: GitHubService.Release,
+        changelog: String
+    ) -> CRPIssue {
+        // [CNSMR-1319] TODO: Use a config file to parametrise accountable person
+        let isTelus = release.appName?.caseInsensitiveCompare("Telus") == .orderedSame
+        let accountablePerson = isTelus ? "eric.schnitzer" : "andreea.papillon"
+        // Remove brackets around JIRA ticket names so that it's recognized by JIRA as a ticket reference
+        // eg replace "[CNSMR-123] Do this" with "CNSMR-123 Do this"
+        let changelog = changelog.replacingOccurrences(of: "\\[([A-Z]+-[0-9]+)\\]", with: "$1", options: [.regularExpression], range: nil)
+        let fields = CRPIssueFields(
+            summary: repoMapping.crp.jiraSummary(release),
+            environments: [repoMapping.crp.environment],
+            release: release,
+            changelog: changelog,
+            accountablePersonName: accountablePerson
+        )
+        return CRPIssue(fields: fields)
+    }
+}
 
-public extension JiraService {
```

Comment:
we don't really need this to be public as it is in the app

Reply:
Yes, it's a remanent of before my own refactor (as it was in the Core before I moved it to the App), I agree we can drop `public` now.